ndbench-web/build.gradle

```diff
@@ -12,6 +12,9 @@ dependencies {
     compile project(':ndbench-es-plugins')
```

Comment:
minor: I would opt to order this list of plugins in alphabetical order, so it is easier to spot missing dependencies in the future.

Reply:
Fixed.